src/visualLayer/components/concert/index.tsx

```diff
@@ -56,9 +57,23 @@ const ConcertForm = ({
       console.log(error);
     }
   };
+  const getAllBrandsList = async () => {
+    try {
+      const resp = await brandsService.getAll();
+      if (resp.success) {
+        // eslint-disable-next-line max-len
+        const brandsOptions = resp.data.brands.map((b) => ({ id: b.id, name: b.name }));
+        setAllBrands(brandsOptions);
+      }
+    } catch (error) {
+      // eslint-disable-next-line no-console
+      console.log(error);
```

Comment:
instead of console log you can show error notification 

Same for above methods